distribution/environments/aws-ec2-cloud/config/as-code/artifact-manager-s3.yaml

```diff
@@ -0,0 +1,15 @@
+---
+# TODO: file JIRAs to make this classifed (and for other used plugins too)
+unclassified:
+  s3blobstoreconfig:
+    # TODO: create through CloudFormation
+    container: "${ARTIFACT_MANAGER_S3_BUCKET_NAME}"
+    # FIXME: not a sensible default for everyone
+    region: "us-east-1"
```

Comment:
Note that this is invalid as of https://github.com/jenkinsci/artifact-manager-s3-plugin/pull/67.